docker/runit/service/spark/run

```diff
@@ -53,6 +53,7 @@ function configure_properties() {
     add_if_non_empty spark.ssl.trustStorePassword "${SPARK_SSL_TRUSTSTOREPASSWORD}"
     add_if_non_empty spark.ssl.protocol "${SPARK_SSL_PROTOCOL}"
     add_if_non_empty spark.ssl.enabledAlgorithms "${SPARK_SSL_ENABLEDALGORITHMS}"
+    echo "spark.mesos.proxy.baseURL=../../mesos" >> /opt/spark/dist/conf/mesos-cluster-dispatcher.properties
```

Comment:
Could you avoid hardcoding the filename (/opt/spark/dist/conf/mesos-cluster-dispatcher.properties)? Maybe store it in a variable?